source/imaer-shared/src/main/java/nl/overheid/aerius/shared/exception/ImaerExceptionReason.java

```diff
@@ -441,6 +441,13 @@ public enum ImaerExceptionReason implements Reason {
    */
   COHESION_REFERENCE_MISSING_BUILDING(5521),
 
+  /**
+   * Buildingheight is < 0.
+   *
+   * @param 0 ID of the building that has height < 0.
+   */
+  BUILDING_HEIGHT_ZERO(5522),
```

Comment:
Most of the 55XY numbers are currently about cohesion. Might make more sense to add this one to the 52XY range?